tools/rec_to_lp/README.md

```diff
@@ -11,7 +11,7 @@ REC files can be retrieved from
 <https://scm.gforge.inria.fr/anonscm/svn/rec>.  The haskell files are
 in `rec/2019-CONVECS/HASKELL/`.  Then use the awk script,
 
-``` shell
+```bash
```

Comment:
``bash`` is an alias for ``shell`` https://github.com/github/linguist/blob/master/lib/linguist/languages.yml